pkgs/top-level/all-packages.nix

```diff
@@ -15615,7 +15615,9 @@ let
     hasktags = haskellPackages.hasktags;
   };
 
-  splix = callPackage ../misc/cups/drivers/splix { };
+  splix = callPackage ../misc/cups/drivers/splix {
+    unstable = config.splix.unstable or false;
```

Comment:
The use of `config.xxx` variables to configure variants inside of Nixpkgs is discouraged. We have the mechanism for historical reasons, but these days we prefer `override` to accomplish that, really.